core/src/types/mod.rs

```diff
@@ -1,3 +1,6 @@
 pub mod aws_config;
 pub mod code;
 pub mod single_or_array;
+
+/// Mappings from legacy `ethers-rs` to `alloy` to facilitate migration.
+pub mod ethers;
```

Comment:
what do you mean by this what is ethers doing in the codebase still?

Reply:
Was meant to represent helper types in the migration for me since I couldn't always find new types in alloy so sometimes for a few structs it was better to just port them in. 

Was meant to be temp, but now I figure they can probably just stay, I can move it to a better filename though